Add details about new workflow to add files in .semgrepignore through App (#799)
returntocorp/semgrep-docs: docs/ignoring-files-folders-code.md

* Semgrep's default `.semgrepignore` file
* Your repository's `.gitignore` file (if it exists)

In the absence of a user-generated `.semgrepignore`, Semgrep refers to [its repository's default template](https://github.com/returntocorp/semgrep/blob/develop/cli/src/semgrep/templates/.semgrepignore):

```

> # Common large paths
> node_modules/
> build/
> dist/
> vendor/
> .env/
> .venv/
> .tox/
> *.min.js
> .npm/
> 
> # Common test paths
> test/
> tests/
> *_test.go
> 
> # Semgrep rules folder
> .semgrep
> 
> # Semgrep-action log folder
> .semgrep_logs/
> 

Author: ```suggestion
DEFAULT_SEMGREPIGNORE_TEXT
```

| Goal | Method |
|:---- |:------ |

> | To scan all files within Semgrep's scope each time you run Semgrep (only files within `.git` are ignored). | Create an empty `.semgrepignore` file in your repository root directory or in your project's working directory. If your `.semgrepignore` file is empty, Semgrep scans only paths in `.gitignore`. |

Reviewer: Is this change here correct? That if the `.semgrepignore` is blank, semgrep will only scan the paths in the `.gitignore`? That doesn't seem right.

Author: Thanks for a good catch @sabrinabrogren! The original is not right as well: `An empty `.semgrepignore` will make Semgrep scan paths in `.gitignore`.` Maybe it means, that when you do not include an empty `.semgreignore` file, Semgrep will use `.gitignore` instead to see what it has to ignore?

Reviewer: That would make sense to me. Let me post this in the #team-app channel, I'm not sure about this.

Author: ```suggestion
| To scan all files within Semgrep's scope each time you run Semgrep (only files within `.git` are ignored). | Create an empty `.semgrepignore` file in your repository root directory or in your project's working directory. An empty `.semgrepignore` will make Semgrep scan paths in `.gitignore`. |
```
